src/components/clusterDetail/ClusterProgress.tsx

```diff
@@ -68,6 +69,19 @@ const getProgressPercent = (hosts: Host[] = []) => {
   return (completedSteps / totalSteps) * 100;
 };
 
+const getOperatorsProgressPercent = (monitoredOperators: MonitoredOperatorsList) =>
+  monitoredOperators.reduce((total, operator) => {
+    let nextTotal = total;
+    if (
+      (operator.operatorType === 'builtin' && operator.status === 'available') ||
+      (operator.operatorType === 'olm' && ['available', 'failed'].includes(operator.status || ''))
+    ) {
+      nextTotal += 1 / monitoredOperators.length;
+    }
+
+    return nextTotal;
+  }, 0);
+
```

Comment:
This seems easier to understand:
```suggestion
export const getOperatorsProgressPercent = (monitoredOperators: MonitoredOperatorsList) => {
  const completeOperators = monitoredOperators.filter(
    (operator) =>
      (operator.operatorType === 'builtin' && operator.status === 'available') ||
      (operator.operatorType === 'olm' && ['available', 'failed'].includes(operator.status || '')),
  );
  return (completeOperators.length / monitoredOperators.length) * 100;
};

```
Also note `* 100` at the end to actually get a percentage.